static/js/emoji.js

```diff
@@ -175,13 +175,17 @@ exports.build_emoji_upload_widget = function () {
     const input_error = $('#emoji_file_input_error');
     const clear_button = $('#emoji_image_clear_button');
     const upload_button = $('#emoji_upload_button');
+    const emoji_preview = $('#emoji_preview_text');
```

Comment:
IMO this should be renamed to `preview_text`.
```suggestion
    const preview_text = $('#emoji_preview_text');
```
Just keeps the naming consistent.